a3p-integration/proposals/n:upgrade-next/package.json

```diff
@@ -18,7 +18,7 @@
   "type": "module",
   "license": "Apache-2.0",
   "dependencies": {
-    "@agoric/synthetic-chain": "^0.1.0",
+    "@agoric/synthetic-chain": "^0.2.1",
```

Comment:
I think master recently bumped this. May be time for a rebase.

Reply:
Resolved in https://github.com/Agoric/agoric-sdk/pull/10229/commits/50a0e17318c083daea6fc2628216adf294525e7a